code/game/jobs/job/job.dm

```diff
@@ -70,6 +70,9 @@
 	///Job Objectives that crew with this job will have a roundstart
 	var/required_objectives = list()
 
+	/// Should this job be allowed to be picked for the bureaucratic error event?
+	var/allow_bureaucratic_error = TRUE
+
```

Comment:
Instead of adding this var, could you make a blacklist of jobs on the bureaucratic error event, and then check if the job is/isn't in that blacklist? Just so we can scope everything to the event datum.

Reply:
Like this?

Reply:
Yeah looks good